corehq/apps/data_pipeline_audit/management/commands/find_sql_forms_not_in_es.py

```diff
@@ -0,0 +1,79 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import unicode_literals
+
+from __future__ import print_function
+from datetime import datetime
+from django.core.management.base import BaseCommand
+import sys
+from django.db.models import Q, F
+from django.db.models.functions import Greatest
+from corehq.form_processor.models import XFormInstanceSQL
+from corehq.apps.es import FormES
+import argparse
+from dimagi.utils.chunked import chunked
+
+DATE_FORMAT = "%Y-%m-%d"
+
+
+def valid_date(s):
+    try:
+        return datetime.strptime(s, DATE_FORMAT)
+    except ValueError:
+        msg = "Not a valid date: '{0}'.".format(s)
+        raise argparse.ArgumentTypeError(msg)
+
+
+class Command(BaseCommand):
+    help = "Print IDs of sql forms that are in the primary DB but not in ES."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
```

Comment:
Should both of these arguments be `required`?

Reply:
Yeah you're right they should be